packages/bitcore-node/src/config.ts

```diff
@@ -62,7 +62,7 @@ const Config = function(): ConfigType {
     dbPass: process.env.DB_PASS || '',
     numWorkers: cpus().length,
     chains: {},
-    modules: ['./bitcoin', './bitcoin-cash', './ethereum'],
```

Comment:
please stash this line, or pull master in.
If you need to enable it, add it to your bitcore.config.json